core/vm/logger.go

```diff
@@ -277,7 +271,7 @@ func (l *StructLogger) CaptureState(pc uint64, op OpCode, gas, cost uint64, scop
 	// Copy a snapshot of the current memory state to a new buffer
 	if l.cfg.EnableMemory {
 		// [Scroll: START]
```

Comment:
All the scroll marks which wrap the reverted lines in `logger.go`, `logger_json.go`, `logger_test.go` should be removed, right?
```suggestion
```

Reply:
That file is in a different state than op-geth as a whole. I'll be aligning everything with another commit.